web-app/js/portal/cart/DownloadPanelItemTemplate.js

```diff
@@ -41,25 +44,48 @@ Portal.cart.DownloadPanelItemTemplate = Ext.extend(Ext.XTemplate, {
             '    </div>',
             '    <div class="floatRight">',
             '      {[this._dataSpecificMarkup(values)]}<br>',
-            '      {[this._shareButtonMarkup(values)]}',
             '    </div>',
             '  </div>',
             '</div>'
         ];
     },
 
+    getTooltip: function(i18nId) {
+        return OpenLayers.i18n(i18nId);
+    },
+
     _getRecordTitle: function(values) {
         return values.title;
     },
 
-    _shareButtonMarkup: function(values) {
-        return String.format(
-            '      <span class="fa fa-fw fa-share-alt fa-lg"></span>' +
-            '      <input readonly onclick="this.focus(); this.select();" title="{0}" value="{1}/home?uuid={2}" />',
-            OpenLayers.i18n('shareButton'),
+    _shareButton: function(collection) {
+        this._createShareButton.defer(1, this, [collection]);
+    },
+
+    _createShareButton: function(collection) {
+        var url =  String.format(
+            '{0}/home?uuid={1}',
             Portal.app.appConfig.grails.serverURL,
-            values.uuid
+            collection.uuid
         );
+        var elementId = this._getLinkId(collection, 'shareButtonId');
+
+        if ( Ext.get(elementId)) {
+
+            Ext.fly(elementId).update("");
+
+            this.shareLink = new Ext.ux.Hyperlink({
```

Comment:
I think this could just be a local variable rather than applying it to `this`. It's not used anywhere else that I can see.